pages/community/events/community-call.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'AsyncAPI Community Call'
+type: COMMUNITY CALL
+description: 'Join us for our upcoming community call to see what’s on our roadmap, early feature demos and product updates from our engineering team.
```

Comment:
```suggestion
description: 'Join our upcoming community call about our roadmap, early feature demos, and engineering updates.
```

It's not really a product, right? it's OSS :D 

and similarly, not really a team 😄, it's a community with engineering contributors